client/src/API/API.ts

```diff
@@ -28,9 +30,265 @@ const logout = async () => {
+      stakeholder: "",
+      scale: "",
+      issuanceDate: null,
+      type: "",
+      connections: "",
+      language: "",
+      pages: null,
+      coordinates: { latitude: 67.8558, longitude: 20.2265 },
+    },
+    {
+      id: 5,
+      title: "Adjusted development plan (47)",
+      description:
+        "This document is the update of the Development Plan, one year after its creation, modifications are made to the general master plan, which is published under the name 'Adjusted Development Plan91,' and still represents the version used today after 10 years. Certainly, there are no drastic differences compared to the previous plan, but upon careful comparison, several modified elements stand out. For example, the central square now takes its final shape, as well as the large school complex just north of it, which appears for the first time.",
+      stakeholder: "",
+      scale: "",
+      issuanceDate: null,
+      type: "",
+      connections: "",
+      language: "",
+      pages: null,
+      coordinates: { latitude: 67.856, longitude: 20.225 },
+    },
+    {
+      id: 6,
+      title: "Document 6",
+      description: null,
+      stakeholder: "",
+      scale: "",
+      issuanceDate: null,
+      type: "",
+      connections: "",
+      language: "",
+      pages: null,
+      coordinates: { latitude: 67.857, longitude: 20.228 },
+    },
+    {
+      id: 7,
+      title: "Document 7",
+      description: "Description 7",
+      stakeholder: "",
+      scale: "",
+      issuanceDate: null,
+      type: "",
+      connections: "",
+      language: "",
+      pages: null,
+      coordinates: { latitude: 67.858, longitude: 20.229 },
+    },
+    {
+      id: 8,
+      title: "Document 8",
+      description: "Description 8",
+      stakeholder: "",
+      scale: "",
+      issuanceDate: null,
+      type: "",
+      connections: "",
+      language: "",
+      pages: null,
+      coordinates: { latitude: 67.859, longitude: 20.23 },
+    },
+    {
+      id: 9,
+      title: "Document 9",
+      description: "Description 9",
+      stakeholder: "",
+      scale: "",
+      issuanceDate: null,
+      type: "",
+      connections: "",
+      language: "",
+      pages: null,
+      coordinates: { latitude: 67.86, longitude: 20.231 },
+    },
+  ];
+}
+
+async function postDocument(id: number, title: string, description: string) {
+  try {
+    const response = await fetch(baseURL + `/documents/${id}`, {
+      method: "POST",
+      credentials: "include",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ title, description }),
+    });
+    if (response.ok) {
+      return true;
+    }
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+async function patchDocument(id: number, title: string, coordinates: string) {
```

Comment:
patch should support a bunch of optional parameters because you may want to edit only some of them. Watch out for nullability this can be done later but leave a //TODO: in the code maybe and fix the type hints

Reply:
For now this API should be use for the connections, indeed if i want to add a new connection to a document i will call this API.

Reply:
to add/edit links you should use PUT /documents/{id}/links. 
Patch /documents/{id} is not used for links (connections)